src/Control/Lens/Internal/ByteString.hs

```diff
@@ -234,41 +215,14 @@ c2w = fromIntegral . ord
 
 -- | Unpack a strict 'B.Bytestring'
 unpackStrict :: B.ByteString -> [Word8]
```

Comment:
Is there a good reason not to just remove this function and replace its use sites with `B.unpack`? I know that this function is exported by this module, but then again, this is an internal module, so we can make breaking changes here without fear.

Reply:
I thought that, but there's also `unpackLazy` (which is also an alias definition `= L.unpack`), so decided to not remove them in this patch.

Reply:
Again, please open an issue about this so that it does not get forgotten.

Reply:
I've done this in #933.